src/core/Song.cpp

```diff
@@ -1150,6 +1151,12 @@ void Song::loadProject( const QString & fileName )
 	m_modified = false;
 	m_loadOnLaunch = false;
 
+	if( isCancelled() )
+	{
+		m_isCancelled = false;
+		createNewProject();
```

Comment:
This should be done earlier. At least before [the error summary](https://github.com/LMMS/lmms/pull/3941/files#diff-bc7aaaf624969ba372535c9eab5ed490R1137).